style.css

```diff
@@ -0,0 +1,94 @@
+html {
+  font-size: 10px;
+}
+
+body {
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+}
```

Comment:
I think you should be more specific with the units of the font sizes.

Reply:
@Fadahunsiseyi Please, mention the reason for your requested changes in your inline comment (e.g: why should the student use a more specific unit with font size?).

Reply:
I think you should be more specific with the unit of the font sizes. This is because they bring uniformity to the code and are better understood by whoever is going to read the code. A specific unit of measurement for the font size should be used across your code. Thanks.